### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
This patch is a collection of cosmetic fixes, nothing mission critical

- https://github.com/RadeonOpenCompute/k8s-device-plugin/commit/9a59e22569dfae528f09e8abb9f9fe13a80625ca updates GO and it's dependencies to 1.16
- https://github.com/RadeonOpenCompute/k8s-device-plugin/commit/9fc3c6e168eb9027a42a197083755a0ddeceb431 Fix Dockerfile formatting and Entrypoint
- https://github.com/RadeonOpenCompute/k8s-device-plugin/commit/fc9251939298ebebe751f1ce827ffc463494a916 Fix markdown lints on README

### CODE REVIEW

**Comment by ArangoGutierrez** [comment]
@y2kenny

File: Dockerfile [hunk 1]
```diff
@@ -26,4 +26,7 @@ RUN apk --no-cache add ca-certificates libdrm
 RUN apk --no-cache add hwloc --repository=http://dl-cdn.alpinelinux.org/alpine/edge/community
 WORKDIR /root/
 COPY --from=0 /go/bin/k8s-device-plugin .
+
 CMD ["./k8s-device-plugin", "-logtostderr=true", "-stderrthreshold=INFO", "-v=5"]
+ENTRYPOINT ["/k8s-device-plugin"]
```
Comment (y2kenny) [inline Dockerfile:31]: Doesn't this create "/k8s-device-plugin ./k8s-device-plugin -logtostderr=true -stderrthreshold=INFO -v=5"

**Review by y2kenny — COMMENTED** [review]
Are you sure this is a correct use of ENTRYPOINT?
https://docs.docker.com/engine/reference/builder/#understand-how-cmd-and-entrypoint-interact

**Comment by ArangoGutierrez** [comment]
Ready for second review @y2kenny

**Comment by ArangoGutierrez** [comment]
> Please separate out the white space and lint fixes and I will merge those first.
> 
> Is the golang update compatible with the supported version of Kubernetes? I recall some compatibility issue in the past w.r.t. golang version but I am not 100% on that.

Done see #22

**Comment by ArangoGutierrez** [comment]
PING @y2kenny

**Comment by ArangoGutierrez** [comment]
PING @y2kenny

**Comment by ArangoGutierrez** [comment]
PING @y2kenny @y2kenny-amd

**Comment by ArangoGutierrez** [comment]
PING @y2kenny @y2kenny-amd

**Comment by ArangoGutierrez** [comment]
PING?

**Pull request closed without merging**